editor/editor_help.cpp

```diff
@@ -2787,19 +2796,21 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, const C
 		} else if (tag == "rb") {
 			p_rt->add_text("]");
 			pos = brk_end + 1;
-		} else if (tag == "url") {
-			int end = bbcode.find_char('[', brk_end);
-			if (end == -1) {
-				end = bbcode.length();
+		} else if (tag.begins_with("url")) {
```

Comment:
This changes the logic, as `urlabc` will now work too.

```suggestion
		} else if (tag == "url" || tag.begins_with("url=")) {
```